### PR TITLE
Removed an unnecessary temp var

### PR DESCRIPTION
**Pull request by technosophos**
This is a follow-up cleanup to an earlier PR.

Signed-off-by: Matt Butcher <matt.butcher@microsoft.com>

### CODE REVIEW

File: src/runtime/mod.rs [hunk 1]
```diff
@@ -364,13 +364,13 @@ impl Module {
         // NB: It is strange that there is not a way to do this already. The Display impl
         // seems to only provide the path.
         let uri = req.uri.clone();
-        let authority = format!("{}:{}", host, port);
         headers.insert(
             "X_FULL_URL".to_owned(),
             format!(
-                "{}://{}{}",
+                "{}://{}:{}{}",
                 uri.scheme_str().unwrap_or("http"), // It is not clear if Hyper ever sets scheme.
```
Comment (thomastaylor312) [inline src/runtime/mod.rs:371]: With https, does this need to be handled differently at all?

**Review by thomastaylor312 — APPROVED** [review]
One non-blocking question, otherwise LGTM

File: src/runtime/mod.rs (hunk 1 above)
Comment (technosophos) [reply src/runtime/mod.rs:371]: Only if Hyper does not set scheme to `https`.

**Review by thomastaylor312 — APPROVED** [review]
New https stuff is looking good

**Pull request merged**